addons/crm/models/crm_lead.py

```diff
@@ -571,7 +571,7 @@ def return_if_relevant(model_name, domain):
             return res if len(res) < SEARCH_RESULT_LIMIT else model
 
         for lead in self:
-            lead_id = lead._origin.id if isinstance(lead.id, models.NewId) else lead.id
+            lead_id = lead._origin.id if isinstance(lead.id, api.NewId) else lead.id
```

Comment:
If `lead` is a real record, then `lead._origin` is the same as `lead`.  Let's simplify the code instead of leaving useless constructions like this.
```suggestion
            lead_id = lead._origin.id
```